base/plumbing/ServerIOBase.js

```diff
@@ -284,7 +284,11 @@ const sogiveid = id => {
 		'water-aid': 'wateraid',
 		'woodland_trust': 'woodland-trust',
 		'woodland':'woodland-trust',
-		'wwf': 'wwf-uk'
+		'wwf': 'wwf-uk',
+		'carers uk':'carers-uk',
+		'Greenwich Food Bank':'greenwich-food-bank',
+		'Camden Food Bank':'camden-food-bank',
+		'Dover Food Bank':'dover-food-bank'
```

Comment:
Hey Ben @dodo721 

Do the lines of code below this in the function not cover these new ones already? 
I'd expect `id.toLowerCase().replace(/[_ ]/g,'-')` to do "Dover Food Bank" > "dover-food-bank".

Dan

Reply:
Hi,
yes they do. That has been left over from some old debugging I did. Woops

Ben